docs/assets/scss/main-content.scss

```diff
@@ -0,0 +1,67 @@
+main {
+  position: relative;
+
+  .td-breadcrumbs {
+    &__single {
+      display: none;
+    }
+
+    .breadcrumb {
+      margin-bottom: 1rem;
+    }
+  }
+
+  .td-content {
+    max-width: 800px;
+    margin: 0 auto;
```

Comment:
I saw that you added that, I think it's quite unusual for docs website to be centered so maybe we should remove the `margin: 0 auto`. And is the `max-width` based on typography rules? I think it's a really nice addition since websites like k8s.io/docs don't have it and might look weird on large screens. We could also use a unit like `ch` to adjust the max-width to an average maximum number of words per line?